feature/exercise-view/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/exerciseview/home/overview/ExerciseOverviewTab.kt

```diff
@@ -24,24 +27,41 @@ internal fun ExerciseOverviewTab(
     webView: WebView?,
     actions: ExerciseActions
 ) {
-    Column(modifier = modifier, verticalArrangement = Arrangement.spacedBy(8.dp)) {
-        Spacer(modifier = Modifier)
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(Color.White),
+        verticalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+
 
         ParticipationStatusUi(
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
             exercise = exercise,
             actions = actions
         )
 
         if (exercise !is QuizExercise && webViewState != null) {
             ArtemisWebView(
-                modifier = Modifier.fillMaxWidth(),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f),
                 webViewState = webViewState,
                 webView = webView,
                 serverUrl = serverUrl,
                 authToken = authToken,
                 setWebView = setWebView
             )
+        } else {
+            Text(
+                text = "No problem statement available.",
```

Comment:
Hardcoding user facing strings will become a problem when trying to localize the app (e.g. into german) later.